### ISSUE TITLE
listports fails on Windows

### ISSUE THREAD

**Issue by bainorama** [pyserial/pyserial#147]
In V3.1.1:
serial.tools.list_ports.comports() fails on a Windows 7 host with:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Python34\lib\site-packages\serial\tools\list_ports_windows.py", line 294, in comports
    return list(iterate_comports())
  File "C:\Python34\lib\site-packages\serial\tools\list_ports_windows.py", line 258, in iterate_comports
    info.hwid = info.usb_info()
  File "C:\Python34\lib\site-packages\serial\tools\list_ports_common.py", line 61, in usb_info
    ' LOCATION={}'.format(self.location) if self.location is not None else '')
TypeError: non-empty format string passed to object.__format__
```
This looks to be caused by the pid/vid parameters not being populated (they're set to None), in list_ports_windows.py, ln 227, szHardwareId_str for the failing case is:
USB\VID_03F0&SUBCLASS_01&PROT_02\7&310FBEA0&0&0000

which doesn't match the regex on ln 228.

**Comment by zsquareplusc** [comment]
I think this is a duplicate of #145 (fix commited: 1532e47)

**Comment by bainorama** [comment]
Yes sorry, looks like it is.
